res/css/views/voip/_CallViewHeader.scss

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_CallView_header {
```

Comment:
This should actually be `mx_CallViewHeader` to match the component name (and so should be the other CSS classes). 

I also tend to prefer the SCSS to be nested which it isn't here but that's up to you

Reply:
In this case I just copied&pasted things so it's easier to read the diff. I can do it now I guess.

Reply:
I am aware it was just copied but it would be best to change now, imo, for consistency and organization

Reply:
Didn't change the nesting, but did change the naming.